src/Entity/User.php

```diff
@@ -90,6 +94,14 @@ class User implements UserInterface, AvailabilitableInterface, UserSerializableI
      */
     public ?Organization $organization = null;
 
+    /**
+     * Organizations whose user is admin.
+     *
+     * @ORM\ManyToMany(targetEntity="App\Entity\Organization", inversedBy="admins")
+     * @ORM\OrderBy({"name"="ASC"})
+     */
+    public Collection $organizations;
```

Comment:
I'm not sure this property name is clear enough. What about `$managedOrganizations` or something else ? Imho `$user->addOrganization()` is not explicit enough